samcli/commands/publish/command.py

```diff
@@ -15,8 +15,9 @@
 
 LOG = logging.getLogger(__name__)
 
-SAM_PUBLISH_DOC = "https://docs.aws.amazon.com/serverless-application-model/latest/developerguide/serverless-sam-template-publishing-applications.html"  # noqa
-SAM_PACKAGE_DOC = "https://docs.aws.amazon.com/serverless-application-model/latest/developerguide/sam-cli-command-reference-sam-package.html"  # noqa
+SAM_DOC_PREFIX = "https://docs.aws.amazon.com/serverless-application-model/latest/developerguide"
+SAM_PUBLISH_DOC = f"{SAM_DOC_PREFIX}/serverless-sam-template-publishing-applications.html"  # noqa
```

Comment:
What was the error here? Doesn't `noqa` prevent the complaints? Feels more of a fix to deal with `pylint`s whims.

Reply:
It was a fix. I will remove `# noqa` here instead of reverting in this case because making the common prefix into a variable does make it easier to maintain